Makefile

```diff
@@ -17,11 +17,11 @@ PACKAGE_NAME := universal-crossplane
 # ====================================================================================
 # Versions
 
-CROSSPLANE_REPO := https://github.com/crossplane/crossplane.git
+CROSSPLANE_REPO := https://github.com/upbound/crossplane.git
 # Tag corresponds to Docker image tag while commit is git-compatible signature
 # for pulling. They do not always match.
-CROSSPLANE_TAG := v1.4.0-rc.0
-CROSSPLANE_COMMIT := v1.4.0-rc.0
+CROSSPLANE_TAG := v1.5.0-rc.0.up.1
+CROSSPLANE_COMMIT := v1.5.0-rc.0-up.1
```

Comment:
Is the dash in `-up.1` intentional? `CROSSPLANE_TAG` has a dot in that place?

Reply:
Yes, this is intentional.
Our build module [converts the latter dash to dot](https://github.com/upbound/build/pull/155) to make the version [sortable as semver](https://github.com/upbound/universal-crossplane/issues/109). This causes upbound/crossplane to produce a docker image with tag `v1.5.0-rc.0.up.1` for git tag `v1.5.0-rc.0-up.1`.

I didn't add additional comments since the existing comment already mentions they do not always match.